package.json

```diff
@@ -31,6 +31,7 @@
     "jsdoc": "jsdoc",
     "prebuild": "node script/prebuild.js",
     "preview": "node script/preview.js",
+    "preview:build": "yarn build:webpack --env.entry=static-pages && yarn preview",
```

Comment:
you would think i'd know this answer but i don't...don't we also need to run `yarn build:content` to run preview?

Reply:
@ncksllvn, isn't it `preview:build` command trying to avoid `yarn build:content`  because it's very slow to rebuild again ?

Reply:
yeah it just seems like a command `preview:build` would do everything you need to get setup for using the preview server

Reply:
I've found that we do, in fact, need to run `yarn build:webpack --env.entry=static-pages` to generate the manifest file _even if_ we've already run `yarn build:content`.